prow/cmd/hook/main.go

```diff
@@ -74,6 +75,7 @@ func gatherOptions() options {
 	flag.StringVar(&o.configPath, "config-path", "/etc/config/config", "Path to config.yaml.")
 	flag.StringVar(&o.cluster, "cluster", "", "Path to kube.Cluster YAML file. If empty, uses the local cluster.")
 	flag.StringVar(&o.pluginConfig, "plugin-config", "/etc/plugins/plugins", "Path to plugin config file.")
+	flag.BoolVar(&o.skipCollab, "skip-collaborators", false, "Skip collaborator checks and use only OWNERS files for approvals+reviews.")
```

Comment:
Can we make this a config option instead of a flag?

Reply:
done